apis/v1alpha1/adminnetworkpolicy_types.go

```diff
@@ -30,6 +30,10 @@ import (
 // +kubebuilder:resource:shortName=anp,scope=Cluster
 // +kubebuilder:printcolumn:name="Priority",type=string,JSONPath=".spec.priority"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+// +kubebuilder:printcolumn:name="Condition",type=string,JSONPath=".status.conditions[*].type"
```

Comment:
I'm not sure if we want to print all of the condition like this in the output... If so we should definitely only print it when `-o wide` is passed 

see `priority=1` for that -> https://github.com/bpfd-dev/bpfd/blob/main/bpfd-operator/apis/v1alpha1/tcProgram_types.go#L34

Reply:
Otherwise things will get very verbose on the command line

Reply:
Why do you need to do that in the command line? 

If you really need ALL of the status info just use a `jsonpath`

Something like 

`kubectl get anp FAKE_ANP  --output="jsonpath={.status.conditions}"`